.github/workflows/test.yml

```diff
@@ -1,11 +1,10 @@
 name: Test
 
-on:
-  - push
-  - pull_request
+on: [push, pull_request]
+
 jobs:
   build:
-    name: build (Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }}, Redis.py ${{ matrix.redis-version }})
+    name: Python ${{ matrix.python-version }}, Django ${{ matrix.django-version }}, Redis.py ${{ matrix.redis-version }}
```

Comment:
I would actually not change this or maybe call it test

Reply:
The title already includes "test" because that's the name of the workflow. You can see what this looks like on this pr

Reply:
I changed this to test because I added back the lint check at your request.